src/lib/pages/backoffice/Acquisition/Order/OrderEditor/OrderForm/OrderForm.js

```diff
@@ -100,10 +104,25 @@ export class OrderForm extends Component {
     return orderApi.update(pid, data);
   };
 
-  successCallback = response => {
-    goTo(
-      AcquisitionRoutes.orderDetailsFor(getIn(response, 'data.metadata.pid'))
+  successCallback = async response => {
```

Comment:
Good job, great UX design.
That said I'm not 100% sure about making the request in the form submit callback. For instance, how should we handle exception? (currently not implemented as fair as I'm aware)

Reply:
If you are talking about exceptions that might occur during adding the provider, I think we can add an action to dispatch an error message instead of directly calling the `addProvider` from the api in the callback.

Reply:
Question for reviewers: I am wondering if this should be done on the backend. After a record creation (signal), update the document request...

Reply:
I'm not sure why it was not done in the backend in the first place. at this point I would say to ticketize it for refatoring @ntarocco WDYT?

Reply:
It is harder than I thought. Let's keep it like this.